src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/casenotes/CaseNoteMapping.kt

```diff
@@ -14,7 +14,7 @@ data class CaseNoteMapping(
 
   val dpsCaseNoteId: UUID,
 
-  val offenderNo: String,
+  var offenderNo: String,
```

Comment:
Ill revert this when the calling client no longer needs it. Currently it is needed for backwards compatibility